addons/test/misra/misra-test.c

```diff
@@ -68,9 +68,33 @@ int misra_5_2_field_hides_field1_31y;//5.2
 };
 const char *s41_1 = "\x41g"; // 4.1
 const char *s41_2 = "\x41\x42";
-const char *s41_3 = "\x41" "g";
+const char *s41_3 = "\x41" "\x42";
+const char *s41_4 = "\x41" "g";
+const char *s41_5 = "\x41\xA";
+const char *s41_6 = "\xA\x41";
+const char *s41_7 = "\xAA\xg\x41"; // 4.1
+const char *s41_8 = "\xAA\x\x41"; // 4.1
+const char *s41_9 = "unknown\gsequence";
+const char *s41_10 = "simple\nsequence";
```

Comment:
```suggestion
const char *s41_10 = "simple\nsequence";
const char *s41_11 = "test";
```
I suggest adding a simple string to the tests to make sure that there is no false positive for such cases in the future.